packages/config/config/devices/0x0109/za1501jp-5.json

```diff
@@ -0,0 +1,39 @@
+// Vision Security ZA1501JP-5
+// Arrival Sensor
+{
+	"manufacturer": "Vision Security",
+	"manufacturerId": "0x0109",
+	"label": "ZA1501JP-5",
+	"description": "Arrival Sensor",
+	"devices": [
+		{
+			"productType": "0x202a",
+			"productId": "0x2a01",
+			"zwaveAllianceId": [3196, 3376]
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": {
+		"1": {
+			"label": "Synchronic Time Interval",
```

Comment:
What is that?

Reply:
@AlCalzone  The best I can come up with is this description from the manual "Define the synchronic interval time between arrival sensor and the gateway"